packages/components/popover/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@contentful/f36-popover",
+  "version": "4.0.0",
+  "private": true,
```

Comment:
don't forget to remove `private`, so the package gets published once merged to the main branch